BINARY_THRESHOLD: 155
NWINDOWS: 20
MINPIX: 50

TL_X: 120
TR_X: 350
BR_X: 390
BL_X: -110

T_Y: 350
B_Y: 480

# T_Y: 355
# B_Y: 433
# T_Y: 402
# B_Y: 480

FALSE_THRES: 2 
STOPLINE_THRES: 120

LANE_DIST: 50

GAMMA: 0.09


